src/PositionManager.sol

```diff
@@ -383,57 +364,78 @@ contract PositionManager is
         if (balance > 0) currency.transfer(to, balance);
     }
 
+    // implementation of abstract function DeltaResolver._pay
+    function _pay(Currency currency, address payer, uint256 amount) internal override {
+        if (payer == address(this)) {
+            // TODO: currency is guaranteed to not be eth so the native check in transfer is not optimal.
```

Comment:
remove the TODO... unless you actually plan to make this change by tuesday?

Reply:
that todo is on main?

Reply:
yeah i know, i just think we should remove it given we're finalising periphery

Reply:
oh i see its literally from this file - how come youve moved it up? the diff confused me

Reply:
esp if we're getting this audited would be good to have a minimal diff